engine/base/runtime/src/main/scala/pl/touk/nussknacker/engine/baseengine/metrics/dropwizard/DropwizardMetricsProviderForScenario.scala

```diff
@@ -13,6 +14,12 @@ class DropwizardMetricsProviderFactory(metricRegistry: MetricRegistry) extends (
   override def apply(scenarioId: String): MetricsProviderForScenario with AutoCloseable = new DropwizardMetricsProviderForScenario(scenarioId, metricRegistry)
 }
 
+object DropwizardMetricsProviderForScenario {
+
+  val scenarioTagName = "process"
```

Comment:
maybe we would change it to `scenario` tag ?

Reply:
We'll do it in next PRs